package.json

```diff
@@ -31,7 +31,7 @@
     "lint-sol": "solium -d contracts/",
     "lint": "yarn run lint-sol && yarn run lint-ts",
     "clean-chain": "rm -rf blockchain && cp -r snapshots/0x blockchain",
-    "chain": "yarn clean-chain && ganache-cli --db blockchain --networkId 50 --accounts 20 -l 10000000 -e 1000000 -m 'concert load couple harbor equip island argue ramp clarify fence smart topic'",
+    "chain": "yarn clean-chain && ganache-cli --db blockchain --networkId 50 --accounts 20 -l 10000000 -e 10000000000 -m 'concert load couple harbor equip island argue ramp clarify fence smart topic'",
```

Comment:
Why is this necessary?

Reply:
Seemed like my chain kept running out of gas